### DISCUSSION TITLE
Errors I got while trying to run the tutorial's example

### DISCUSSION THREAD

**Discussion by ivarismartinez** [alphaparrot/ExoPlaSim#7] [Q&A]
Hello, does anyone know what could be causing these errors? I am using WSL for Windows 11. 
![Screenshot 2023-07-18 151604](https://github.com/alphaparrot/ExoPlaSim/assets/135998966/14d22e27-6e7e-4ed9-b57a-e2a4b76066d1)

**Comment by alphaparrot** [comment]
Hi, this looks to me like either gfortran or openmpi are not
installed/configured correctly in the WSL. ExoPlaSim is failing to find a
compiler.

**Comment by ivarismartinez** [comment]
Thanks for the reply!

Respectfully, Ivaris C. Martínez Serrano
